docs/checkedc/Test-Baselines.md

```diff
@@ -52,6 +89,16 @@ Here is the current base line for testing LLVM + clang on all targets (check-all
            Unexpected Failures: 6
 ```
 
+## Delta between the master branch and baseline branch
+
+We have added tests for Checked C to the master branch.  These tests are specific to clang, such as tests of
+compiler internals or the driver.  We currently expect the master branch to pass the following additional tests
+in these configurations.
+
+- For just clang with the x86 target (using the check-clang project), 3 additional `Expected Passes` tests.
+- For testing LLVM + clang on x86 (check-all), 8 additional `Expected Passes` tests
```

Comment:
Agreed.  Updating precise counts will likely become a developer burden.  We only need to track 'Unexpected failures' in the baseline branch.  We don't particularly need to track the counts. I'll update the md files in the pull request to reflect that. 

I did have some concerns that people will add tests incorrectly and not notice they aren't running.   However, this seems pretty unlikely given the way that the LLVM test infrastructure  works.  It will produce an error for tests that are lacking RUN lines.  The only requirement during code reviews to make sure that tests will run is to make sure they are placed within test suite directories.